### DISCUSSION TITLE
Avaible for Linux ?

### DISCUSSION THREAD

**Discussion by alberic89** [n00mkrad/flowframes#152] [Ideas]
(excuse the mistakes, I am French)
How I can build this app in Linux ? Can you create a .deb package, or other (snap, flatpack), for the linux users ?

**Comment by Sturmlocke86** [comment]
+1 I was just checking out their itch.io website to see if I can purchase this on Linux. I'd purchase it for Windows and run it through Wine if that works? I'll pay up to 30 Euros for a proper Linux port.

**Comment by Sturmlocke86** [comment]
In my search for something compatible with Linux, I found this: https://github.com/TNTwise/REAL-Video-Enhancer

It's basically just like Flowframes in many ways. The devs are super friendly and very responsive too. I would still purchase a Linux version of Flowframes, and would donate to Real Video Enhancer at the same time. Nothing wrong in supporting both projects. We need more apps like these on Linux ideally easily accessible + user friendly via Flathub and Snap. Ty

**Comment by Dmole** [comment]
ffmpeg minterpolate for blurry results.
